src/tv2-common/inewsConversion/converters/ParseCue.ts

```diff
@@ -722,18 +722,23 @@ function parseTargetEngine(
 	}
 
 	const graphicConfig = config.showStyle.GFXTemplates.find(
-		tmpl =>
-			tmpl.INewsCode.toUpperCase() === code?.toUpperCase() && tmpl.INewsName.toUpperCase() === iNewsName.toUpperCase()
+		template =>
+			template.INewsCode.toUpperCase() === code?.toUpperCase() &&
+			template.INewsName.toUpperCase() === iNewsName.toUpperCase()
 	)
 
+	const graphicDesignConfig = code
+		? config.showStyle.GfxDesignTemplates.find(template => template.INewsName.toUpperCase() === code?.toUpperCase())
+		: undefined
+
 	if (graphicConfig) {
 		if (!!graphicConfig.VizTemplate.toUpperCase().match(/^VCP$/i)) {
 			engineCue.mergeable = true
 		} else {
-			if (graphicConfig.IsDesign) {
+			if (graphicDesignConfig) {
```

Comment:
Will we ever reach this block after the design templates were moved into a separate table? I don't know how this would be possible in the first place (because I don't know all the possible usages of the target cue), but if `iNewsName` were to be a name of a certain design, then `graphicConfig` probably will be undefined, yet `graphicDesignConfig` will have the template we're looking for.

Reply:
You're right! I will look into this friday on friday!

Reply:
Rearranged the statements so it would be reachable in [3b084c1](https://github.com/tv2/sofie-blueprints-inews/pull/153/commits/3b084c140ddef3c5ba4bc5c2c862edd1ebab1415)!